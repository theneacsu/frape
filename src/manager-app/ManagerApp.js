import React from 'react'
import TodoApp from './todo-app/TodoApp'
import { Provider } from 'react-redux'
import configureStore from '../store/configureStore'

const store = configureStore()

const ManagerApp = () => (
  <div>
    <Provider store={store}>
      <TodoApp />
    </Provider>
  </div>
)

export default ManagerApp
