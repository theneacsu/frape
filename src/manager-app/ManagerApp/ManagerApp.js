import React from 'react'
import AppRouter from '../../router/AppRouter'
import styles from './ManagerApp.module.css'

const ManagerApp = () => (
  <div className={styles.div}>
    <AppRouter />
  </div>
)

export default ManagerApp
