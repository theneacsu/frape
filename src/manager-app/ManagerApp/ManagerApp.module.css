* {
  box-sizing: border-box;
}

html {
  font-size: 62.5%;
  margin: 0;
  background-color: #dddcd8;
}

body {
  color: #47667c;
  font-family: Helvetica, Arial, sans-serif;
  line-height: 1.6;
  font-size: 1.6rem;
}

.div {
  display: flex;
  flex-flow: column;
  min-height: 100vh;
}

button {
  cursor: pointer;
  outline: none;
}

input,
textarea {
  outline: none;
}

:global(.sm-bm) {
  margin-bottom: 2rem;
}
