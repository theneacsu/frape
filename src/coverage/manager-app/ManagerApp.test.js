import React from 'react'
import { shallow } from 'enzyme'
import ManagerApp from '../../manager-app/ManagerApp'

test('Should render correctly the ManagerApp component', () => {
  const wrapper = shallow(<ManagerApp />)
  expect(wrapper).toMatchSnapshot()
})
