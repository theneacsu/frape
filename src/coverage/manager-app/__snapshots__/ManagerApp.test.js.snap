// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly the ManagerApp component 1`] = `
<div>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <TodoApp />
  </Provider>
</div>
`;
